### PR TITLE
Feat product page

### PR DESCRIPTION
**Pull request by ankit9015**
This PR has following features:
1. Product backend is updated
2. Updated Cards file to adjust for ProductsCard
3. Had to update home page file due to the changes in Cards component
4. card.css is adding additional styling over the component library styles for precise styling requirements
5. some products card are without image will update the actual product data later

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -1,16 +1,15 @@
 import React from "react";
-import {Link} from "react-router-dom"
+import { Link } from "react-router-dom";
 
 const CardButton = (props) => {
```
Comment (JantuDeb) [inline src/components/Card/Card.jsx:4]: You can destructure the props here

File: src/components/Card/ProductCard.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { CardImage, CardButton } from "./Card";
+import "./card.css";
+
+function ProductCard(props) {
+  return (
+    <div class="card-vertical in-wishlist">
+      <CardImage src={props.img.src} alt={props.img.alt} />
+      <div class="badge-wishlist flex-row button">
```
Comment (JantuDeb) [inline src/components/Card/ProductCard.jsx]: same here

File: src/pages/Products/DisplaySection.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+      <div class="product-card-container">
+        {products.map((product) => (
+          <ProductCard
+            name={product.name}
+            discount={product.discount}
+            discountedPrice={product.discountedPrice}
+            price={product.price}
+            ratings={product.ratings}
+            img={{ src: product.img, alt: product.name }}
```
Comment (JantuDeb) [inline src/pages/Products/DisplaySection.jsx]: Instead of passing these individually you can pass `product={product}`

**Review by JantuDeb — APPROVED** [review]

File: src/components/Card/Card.jsx [hunk 4]
```diff
@@ -19,34 +18,37 @@ const CardImage = (props) => {
...
-        {props.overlayText && 
-        <div className="text-overlay text-center H2">{props.overlayText}</div>}
+    <div
+      className={`card-shadow ${
+        props.variant === "vertical" ? "card-vetical" : "card-horizontal"
+      } ${props.overlayText ? "text-overlay-container" : ""}
+        ${props.className ? props.className : ""}`}
+    >
+      <CardImage src={props.img.src} alt={props.img.alt} />
+
```
Comment (rkapoor10) [inline src/components/Card/Card.jsx:28]: you can destructure props

File: src/components/Card/ProductCard.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      <div className="card-body">
+        <h1 className="card-title">{props.product.name}</h1>
+
+        <div className="card-description">
+          <div className="text-md">
+            {`Ratings: ${props.product.ratings}`}
+            <i className="rating-star fas fa-star"></i>
+          </div>
+          <span className="price-new text-lg font-extrabold">
+            {`Price: ₹${props.product.discountedPrice}`}{" "}
```
Comment (rkapoor10) [inline src/components/Card/ProductCard.jsx:21]: remove empty {" "}

File: src/components/Card/ProductCard.jsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+          </span>{" "}
```
Comment (rkapoor10) [inline src/components/Card/ProductCard.jsx:22]: here also

File: src/pages/Products/DisplaySection.jsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { products } from "../../backend/db/products";
+import ProductCard from "../../components/Card/ProductCard";
+
+export function DisplaySection() {
+  return (
+    <div className="product-display-section">
+      <h2 class="H2 font-bold line-height-lg">
+        Showing All Products{" "}
```
Comment (rkapoor10) [inline src/pages/Products/DisplaySection.jsx:9]: here also

File: src/pages/Products/SideBar.jsx [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+          <input type="radio" name="rating-filter" id="equal-and-above-3" />
+          <label for="equal-and-above-3">3 stars and above</label>
+        </div>
+        <div>
+          <input type="radio" name="rating-filter" id="equal-and-above-2" />
+          <label for="equal-and-above-2">2 stars and above</label>
+        </div>
+        <div>
+          <input type="radio" name="rating-filter" id="equal-and-above-1" />
+          <label for="equal-and-above-1">1 stars and above</label>
```
Comment (rkapoor10) [inline src/pages/Products/SideBar.jsx:76]: you can use map here to avoid repeated jsx

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge...just go through the suggestions once

**Pull request merged**